es/webapp/src/main/webapp/i18n/vivo_all_es.properties

```diff
@@ -214,6 +214,8 @@ define_value_for_property = Para utilizar esta función, defina un valor para es
 faculty_capitalized = Facultad
 loading_faculty = Cargando facultad. . .
 research_capitalized = Investigación
+items_of_type = Objetos de tipo
+thousands_short = k
 view_all = Ver todo ...
 view_all_research = Ver todas las investigaciones
 no_research_content_found = No se encontró contenido de investigación.
```

Comment:
No se ha encontrado contenido de investigación.

Reply:
Fixed. Thank you.